rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -7970,3 +7971,98 @@ fn subnet_info_ingress_fails() {
             "cannot be called by a user",
         );
 }
+
+#[test]
+fn node_metrics_history_update_succeeds() {
+    let own_subnet_id = subnet_test_id(1);
+    let mut test = ExecutionTestBuilder::new()
+        .with_own_subnet_id(own_subnet_id)
+        .build();
+    let uni_canister = test
+        .universal_canister_with_cycles(Cycles::new(1_000_000_000_000))
+        .unwrap();
+    let update = wasm()
+        .call_simple(
+            CanisterId::ic_00(),
+            Method::NodeMetricsHistory,
+            call_args().other_side(
+                NodeMetricsHistoryArgs {
+                    subnet_id: own_subnet_id.get(),
+                    start_at_timestamp_nanos: 0,
+                }
+                .encode(),
+            ),
+        )
+        .build();
+    let result = test.ingress(uni_canister, "update", update);
+    let bytes = get_reply(result);
+    let _ = Decode!(&bytes, Vec<NodeMetricsHistoryResponse>).unwrap();
+}
+
+#[test]
+fn node_metrics_history_query_fails() {
```

Comment:
I suppose this should be `node_metrics_history_ingress_query_fails` since it's doing an ingress query call.